conf/sbc_sharc.config

```diff
@@ -0,0 +1,59 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+              
+              Sheffield Bioinformatics Core Configuration Profile - ShARC
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+          
+                            Base Institutional Configuration
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+
+params {
+  
+  // nf-core specific parameters displayed in header summary of each run
+
+  config_profile_description = 'Sheffield Bioinformatics Core - ShARC'
+  config_profile_contact = 'Lewis Quayle (l.quayle@sheffield.ac.uk)'
+  config_profile_url = 'https://docs.hpc.shef.ac.uk/en/latest/sharc/index.html'
+  
+  // hpc resource limits
+
+  max_cpus = 16
+  max_memory = 64.GB
+  max_time = 96.h
+
+}
+
+
+// container engine
+
+singularity {
+
+  enabled = true
+  autoMounts = true
+  // cacheDir = '/<path>/<to>/<shared>/<image_cache>'
+
+}
+
+
+// hpc configuration specific to ShARC
+
+process {
+
+  // scheduler
+
+  executor = 'sge'
+  penv = 'smp'
+  queue = { task.time <= 6.h ? 'shortint.q' : 'all.q' }
+  clusterOptions = { "-l rmem=${task.memory.toGiga()}G" }
+
+  // error and retry handling
+
+  errorStrategy = { task.exitStatus in [143,137,104,134,139,140] ? 'retry' : 'finish' }
+  maxRetries = 2
```

Comment:
I would normally leave this on a per-pipeline basis, as exit codes can somtimes mean different things to different tools